### DISCUSSION TITLE
Allow to open links on README.md page in a new tab

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [community/community#5089] [Projects and Issues]
Right now all links in README.md open in the same tab.
Can you guys please add the ability to open links in a new tab by wrapping them with:

```html
<a target="_blank" href="https://link">
   ...
</a>
```

**Comment by jaredmcateer** [comment]
I am not a fan of pages hijacking my control. If I want a link to open in a new tab I can middle mouse click, <kbd>mmb</kbd>, or  <kbd>ctrl</kbd>+<kbd>click</kbd>, but there is no easy way without a browser extension or injecting user scripts to make a link open on the same page.

**Reply by aleksey-hoffman** [reply]
Well, on the other hand, most developers want to have control over their readme page.

For example, on my app's readme page I have quick links to other project's pages and 3rd party websites like Patreon. So if a person clicks on a Patreon link, navigates around there and signs up on their website, they would need to click the back button multiple times to get back. Not great for UX.

In some cases it even decreases user retention rate, because some people get distracted easily and instantly forget what they were doing a minute ago and why they clicked on that link, which often results in them just closing the tab and leaving your project page.

**Reply by jaredmcateer** [reply]
> So if a person clicks on a Patreon link, navigates around there and signs up on their website, they would need to click the back button multiple times to get back. Not great for UX.

Most people, especially the types to be viewing a github repository, already know how to open links in new tabs if they desire. 

If I don't open a link into a new tab I likely don't have intention of coming back, but if I do I always have back history which I can easily skip multiple pages by accessing the tab history with a <kbd>right</kbd>+<kbd>click</kbd> (or long press for mobile) on the back button.

> In some cases it even decreases user retention rate, because some people get distracted easily and instantly forget what they were doing a minute ago and why they clicked on that link, which often results in them just closing the tab and leaving your project page.

Here is an article by Niesel Normal Group that addresses this specific point and the whole topic in general. https://www.nngroup.com/articles/new-browser-windows-and-tabs/

> **It will keep users on our site.**
> * By gratuitously adding an extra window or tab, you are increasing users’ information-management burden.
> * Users will have to switch to the new window or tab to complete their task and there’s no guarantee that they will return to your site afterwards.
> * People are not dumb: they will realize the intent behind the extra windows and tabs, and may start to distrust your site and brand.
> * If people want to leave a website, application, or document, they'll do so on their own terms.

I think points 1,2 and 4 are most relevant. Just because you have avoided letting the user navigate away from your page it doesn't necessarily mean they want to stay, rather it is another extra step necessary to close the tab. Would it not be a better UX to let them to be able to do what they want?

**Reply by aleksey-hoffman** [reply]
Your points are valid. Maybe it is better to have the current model after all
